### PR TITLE
15.0 adding new module

### PR DESCRIPTION
**Pull request by tatianarosabal**

### CODE REVIEW

File: roto_accounting/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2022-TODAY Rapsodoo Iberia S.r.L. (www.rapsodoo.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+{
+    'name': "Roto Accounting",
+    'summary': 'Changes to be consider in Roto CLub Odoo enviroment',
+    'author': "Rapsodoo Iberia",
+    'website': "https://www.rapsodoo.com/es/",
+    'category': 'Accounting/Accounting',
+
```
Comment (supportiberia) [inline roto_accounting/__manifest__.py:10]: Delete empty space, please!!!!!!

File: roto_accounting/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    'version': '15.0.1.0.0',
```
Comment (supportiberia) [inline roto_accounting/__manifest__.py:11]: Version = 15.0.0.0.1?

File: roto_accounting/models/analytic_group_inherit.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    _inherit = 'account.analytic.group'
+
+    partner_id = fields.Many2one(
+        comodel_name='res.partner',
+        string='Shipping'
+    )
+    address = fields.Char(
+        string='Address'
+    )
+
```
Comment (supportiberia) [inline roto_accounting/models/analytic_group_inherit.py:17]: Empty space

**Review by supportiberia — COMMENTED** [review]
Review your code please

File: roto_accounting/__manifest__.py (hunk 2 above)
Comment (tatianarosabal) [reply roto_accounting/__manifest__.py:11]: the first version is always 1.0.0

**Review by supportiberia — APPROVED** [review]

**Pull request merged**